spec/requests/bulk_submissions_controller_spec.rb

```diff
@@ -147,4 +147,30 @@
         .or redirect_to(authenticated_root_path)
     end
   end
+
+  describe "GET /download" do
+    before { bulk_submission }
+
+    let(:bulk_submission) do
+      create(:bulk_submission,
+             :undiscarded,
+             :pending,
+             :with_original_file,
+             :with_result_file,
+             user_id: user.id)
+    end
+
+    it "downloads the results file" do
+      get download_bulk_submission_path(bulk_submission.id)
+      expect(response)
+        .to redirect_to(redirect_to(rails_blob_path(bulk_submission.result_file.blob, disposition: 'attachment')))
+    end
+
+    it "logs the user downloading the results file" do
+      allow(Rails.logger).to receive(:info).and_call_original
```

Comment:
i guess you set test to :warn in order to silence the log output for this test? but if that is the case then I am not seeing any output in any event even when i set back to :debug - the default for test). it only impacts log output at `log/test.log`